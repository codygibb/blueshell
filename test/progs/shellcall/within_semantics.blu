within ("~/dir1/dir2") {
  print "inside within"
}
print "outside within"

func a(x, y) {
  return str(x + y)
}

within a(1, 2) {
  print "inside within again"
}
