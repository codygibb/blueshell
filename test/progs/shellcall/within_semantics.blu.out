inside within
outside within
inside within again
